client/src/components/Ratings/Review/Helpfulness.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+export default function Helpfulness({
+  addHelpVote, helpfulness, reportReview, reviewId, index,
+}) {
+  const [helpCount, setHelpCount] = useState(helpfulness);
+  const [voteLimiter, setvoteLimiter] = useState(true);
+
+  const vote = () => {
+    if (voteLimiter) {
+      addHelpVote(reviewId);
+      setHelpCount(helpCount + 1);
+    }
+    setvoteLimiter(voteLimiter && false);
+  };
+
+  const report = () => {
+    reportReview(index, reviewId);
+  };
+  return (
+    <HelpfulnessContainer>
+      Helpful?
+      <FontLikeButton onClick={vote}>
+        Yes
+      </FontLikeButton>
+      <WhiteSpaceWrapper>
```

Comment:
Minor nitpick but I might call this `HelpCount` instead

Reply:
Done